Kickstarter-iOS/Views/Cells/EmptyCommentsCell.swift

```diff
@@ -0,0 +1,60 @@
+import KsApi
+import Library
+import Prelude
+import UIKit
+
+final class EmptyCommentsCell: UITableViewCell, ValueCell {
+  // MARK: - Properties
+
+  private lazy var noCommentsLabel: UILabel = { UILabel(frame: .zero) }()
+
+  fileprivate let viewModel: EmptyCommentsCellViewModelType =
+    EmptyCommentsCellViewModel()
+
+  // MARK: - Lifecycle
+
+  override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+    super.init(style: style, reuseIdentifier: reuseIdentifier)
+
+    self.bindStyles()
+    self.bindViewModel()
+    self.configureViews()
+  }
+
+  required init?(coder aDecoder: NSCoder) {
+    super.init(coder: aDecoder)
+  }
+
+  // MARK: - Styles
+
+  override func bindStyles() {
+    super.bindStyles()
+
+    _ = self
+      |> baseTableViewCellStyle()
+
+    _ = self.noCommentsLabel
+      |> \.lineBreakMode .~ .byWordWrapping
+      |> \.numberOfLines .~ 0
+      |> \.adjustsFontForContentSizeCategory .~ true
+      |> \.font .~ UIFont.ksr_callout()
+  }
+
+  // MARK: - Configuration
+
+  internal func configureWith(value: Project) {
```

Comment:
Do we need to configure this with `Project`? You should be able to make the type `Void` here and still be able to conform to `ValueCell`.